drivers/overlay/overlay.go

```diff
@@ -677,6 +677,40 @@ func (d *Driver) Remove(id string) error {
 	return nil
 }
 
+// recreateSymlinks goes through the driver's home directory and checks if the diff directory
+// under each layer has a symlink created for it under the linkDir. If the symlink does not
+// exist, it creates them
+func (d *Driver) recreateSymlinks() error {
+	// List all the directories under the home directory
+	dirs, err := ioutil.ReadDir(d.home)
+	if err != nil {
+		return fmt.Errorf("error reading driver home directory %q: %v", d.home, err)
+	}
+	for _, dir := range dirs {
```

Comment:
Please screen out non-directories here.  If we turn on quota enforcement, other things show up in this directory, and we don't want to trip over them.

Reply:
Fixed